tests/benchmark/snappy/tpcds/PerfRun.conf

```diff
@@ -0,0 +1,60 @@
+#Do not change the way of queries, sparkProperties, sparkSqlProperties. Just change the values inside strings
```

Comment:
Before running the perf, we create a PerfRun.conf from the PerfRun.conf.template, so this file can be deleted.